views/getinvolved.tmpl

```diff
@@ -110,30 +108,34 @@
             <input type="text" id="phone" name="phone" class="form-control">
           </div>
           <div class="form-group">
-              <label for="college">College</label>
-              <select id="college" name="college" class="form-control">
-                <option value="10">Unknown</option>
-                <option value="1">Alcuin</option>
-                <option value="11">Constantine</option>
-                <option value="2">Derwent</option>
-                <option value="3">Goodricke</option>
-                <option value="4">Halifax</option>
-                <option value="5">James</option>
-                <option value="6">Langwith</option>
-                <option value="7">Vanbrugh</option>
-                <option value="8">Wentworth</option>
-              </select>
+            <label for="college">College</label>
+            <select id="college" name="college" class="form-control">
+              <option value="10">Unknown</option>
```

Comment:
Hmm, should this list be hardcoded?

Reply:
I agree that it shouldn't but I left it as I found it (only looks like I changed it because indentation). 
Will change it if poissible